ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -77,4 +93,4 @@ int
 gcp_connection::bucket_exists(bool &exists) const
 {
     return 0;
-}
+}
```

Comment:
add back the new line

Reply:
addressed here: [600b09f](https://github.com/wiredtiger/wiredtiger/pull/8717/commits/600b09f6e6320c6b3a271bf48c7bbe49933a1788)